src/index.ts

```diff
@@ -2294,7 +2295,7 @@ export class GossipSub extends TypedEventEmitter<GossipsubEvents> implements Pub
       topic,
       tosendCount,
       tosend.size,
-      rawMsg.data != null ? rawMsg.data.length : 0,
+      RPC.Message.getSize(rawMsg),
```

Comment:
I noticed this, we were only counting the "data" portion but more than that gets sent over the wire in a `RPC.message`.  I updated with this method but not sure if that is "correct" because I did not find anything in the spec speaking to this.